cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -1216,6 +1231,32 @@ features:
         projectId: Project ID
         serviceAccountKeyfile: Service account key file location
         testConnection: Test Connection
+      ADLS:
```

Comment:
nit: This section should be in alphabetical order. 

```
ADLS:
  accountFQDN: ...
  Buttons:
    ...
  clientID: ...
```

Reply:
Arranged in alphabetical order.